### PR TITLE
Update ISSUE_TEMPLATE.md

### LINKED ISSUE #526: Add fields for python and pip versions along with OS details in the issue template.
### I'm submitting a 
- [ ] bug report.
- [x] feature request.

### Current Behaviour:

Without knowing exactly the python, pip and OS details, it's very difficult to reproduce the errors. 
### Expected Behaviour:

Having them in the issue template will help to reproduce bugs faster. 
### Do you want to work on this issue?

yes

### PR DESCRIPTION
**Pull request by Purvanshsingh**
Fixes #526

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.6.0 and above.

### Description

Configurations field added to the ISSUE_TEMPLATE.md

#### Added 
python version
pip version
OS Details

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -11,5 +11,11 @@
 ### Steps to reproduce:
 <!-- If you can then please provide the steps to reproduce the bug -->
 
+### Configurations:
```
Comment (priyanshunayan) [inline .github/ISSUE_TEMPLATE.md:14]: Is "Configurations" a valid English word? Can we not use "Environment" here instead?

**Review by priyanshunayan — CHANGES_REQUESTED** [review]
Just a minor change required.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Purvanshsingh) [reply .github/ISSUE_TEMPLATE.md:14]: I thought of "Environment" before but was not sure it will be appropriate or not.
will change it to "Environment".

**Review by priyanshunayan — APPROVED** [review]
LGTM 👍

**Pull request merged**